### DISCUSSION TITLE
Add ability to export .deb and .rpm

### DISCUSSION THREAD

**Discussion by invy** [microsoft/vcpkg#17833] [Ideas]
vcpkg contains all needed dependency information and is built in the similar way as linux package managers are.
vcpkg also already has similar export (#6891 )
It would great if vcpkg could export set of inter-dependent linux packages (deb, rpm, ...).
Later these packages could be used in custom package source and installed in the custom tailored distribution via native package manager.

**Comment by martinboers** [comment]
+1
And packages in .ipk format, please.
